snapcraft/tests/unit/project_loader/test_config.py

```diff
@@ -909,6 +858,67 @@ def load_effect(*args, **kwargs):
         self.mock_load_part.assert_has_calls([call1, call2], any_order=True)
 
 
+class ValidVersionTestCase(YamlTestCase):
+    scenarios = [
+        (version, dict(version=version)) for
+        version in ['buttered-popcorn', '1.2.3', 'v12.4:1:2~', 'HeLlo']
+    ]
+
+    def test_valid_yaml_invalid_version(self):
```

Comment:
This should probably be called `test_valid_yaml_valid_version`? Or even `test_valid_version`? I find these double valid/invalid rather confusing personally so I like the shorter name more.

Reply:
Yes these are silly indeed, thank you. Fixed.